Header header
geometry_msgs/Vector3 position
geometry_msgs/Vector3 velocity
float64 yaw
